src/main/resources/templates/progress/getHintList.html

```diff
@@ -28,7 +28,9 @@ <h2>Dostupné hinty pro Šifru <span th:text="${cypher.name}"></span></h2>
                 <td id="hintId" align="center" th:text="${hint.hintId}">value</td>
                 <td align="center" th:text="${hint.value}">value</td>
                 <td align="center">
-                    <button th:onclick="'javascript:openDialog(' + ${hint.hintId} + ')'">Vzít Hint</button>
+                    <button th:disabled="${takenHints.get(hint.hintId) != null}"
```

Comment:
Toto je validni? Muzeme mit v DB takove data? Pokud ano, tak bychom meli vyresit spis pricinu, jak muzou vzniknout.

Reply:
Ja doufam ze jo.
` model.addAttribute("takenHints", progressService.setTakenHintsToMap(cypher, team));`
takenHints je mapa, ktera v sobe ma vsechny hinty, ktere si tym pro jednotlivou sifru vzal.
Pokud si tym zadny hint nevzal, tak takenHints.get(hintId) je prazdna.

Reply:
tim jsem chtel rict, ze to netaha z databaze :)